specs/jointcal/hsc_r.yaml

```diff
@@ -0,0 +1,42 @@
+# Specifications for jointcal metrics against testdata_jointcal
+# hsc r-band datasets.
+
+---
+id: 'base'
+provenance_query:
+  # Prototype for query provenance. This needs to be figured out.
+  dataset_repo_url: 'https://github.com/lsst/testdata_jointcal.git'
+
+---
+# jointcal.astrometry_ndof.hsc_11_visits
+name: "hsc_11_visits"
+  base: ["#astrometry_ndof-base", "#base"]
```

Comment:
Thanks. I've cleaned that up, and touched up the rest of this file too. I think these specifications are going to have to remain placeholders until we have Specification working, with a linter.